extension/extend-admin-console-spi/README.md

```diff
@@ -0,0 +1,36 @@
+## Extending the admin ui with SPI
+
+This example shows how you can extend the admin ui by implementing the SPI.
+You can either add a tab to an existing page or have a complete section in the menu.
+By implementing either:
+
+  * `org.keycloak.services.ui.extend.UiPageProvider` for a complete section
+  * `org.keycloak.services.ui.extend.UiTabProvider` for a tab
+
+For both options you need to describe the fields you want on the page by using the `org.keycloak.provider.ProviderConfigProperty`
+To specify where the tab will appear in the ui, you can add a path valid paths can be found in the `routes.ts` files.
+
+This example contains an implementation of both these `AdminUiPage` is a todo section that is added.
+And `ThemeUiTab` is an extra tab in the realm setting that let's you edit the realm attributes.
+
+### Running
+
+Run this example by compiling with maven:
+
+```bash
+$ mvn install
```

Comment:
All the other tests have the following command to skip tests (`-DskipTests=true`) and also to take the compilation steps from the main pom (`-Pextension`).
```suggestion
mvn -Pextension clean install -DskipTests=true
```
Is it done here differently on purpose?